taskcluster/tc-train-tests.sh

```diff
@@ -8,6 +8,13 @@ extract_python_versions "$1" "pyver" "pyver_pkg" "py_unicode_type" "pyconf" "pya
 
 bitrate=$2
 
+decoder_src=$3
+
+if [ "$decoder_src" = "--pypi" ]; then
```

Comment:
nit: `--pypi` is confusing, I was expecting the code to indeed fetch from pypi, can we change to `--no-pypi` ?

Reply:
That's what it does, fetch from PyPI. I guess the code is even more confusing then. What's leading you to believe that this flag is "disabling PyPI"?

Reply:
Commenting after waking up, I guess ...